basetest.pm

```diff
@@ -17,6 +17,13 @@ use Mojo::File 'path';
 my $serial_file_pos = 0;
 my $autoinst_log_pos = 0;
 
+my $fail_severity = {
```

Comment:
This hash probably fits in one line and you can avoid quoting the keys.

Reply:
The list just got longer. Also putting it in one line would make it hard to read.
Regarding the quoting I would rather use quoting and not go too deep into the perl rabbit hole (also grepping for quoted strings is easier that way).